pkg/core/transaction/attrtype.go

```diff
@@ -14,11 +14,12 @@ const (
 
 // List of valid attribute types.
 const (
-	HighPriority    AttrType = 1
-	OracleResponseT AttrType = 0x11 // OracleResponse
-	NotValidBeforeT AttrType = 0x20 // NotValidBefore
-	ConflictsT      AttrType = 0x21 // Conflicts
-	NotaryAssistedT AttrType = 0x22 // NotaryAssisted
+	HighPriority         AttrType = 1
+	OracleResponseT      AttrType = 0x11 // OracleResponse
+	NotValidBeforeT      AttrType = 0x20 // NotValidBefore
+	ConflictsT           AttrType = 0x21 // Conflicts
+	NotaryAssistedT      AttrType = 0x22 // NotaryAssisted
+	RefundableSystemFeeT AttrType = 0x30 // RefundableSystemFee
```

Comment:
String representation of attribute type need to be updated.

Reply:
done

Reply:
Could you also, please, squash the commits?

Reply:
We can wait for a bit more discussion in neo-project/neo, either it becomes an extension or a part of the proper protocol. Then tune/squash/whatever.

Reply:
Then I will squash when it's ready.